base/binaryplatforms_pkgcompat.jl

```diff
@@ -0,0 +1,150 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+"""
+    Base.BinaryPlatforms.PkgCompat is the old BinaryPlatforms API exposed by
+    Pkg.jl as Pkg.BinaryPlatforms
```

Comment:
```suggestion
Base.BinaryPlatforms.PkgCompat is the old BinaryPlatforms API exposed by
Pkg.jl as Pkg.BinaryPlatforms
```
This is prose but indenting by 4 spaces will make it render as a code block.

Reply:
Resolve in [bca6b82](https://github.com/JuliaLang/julia/pull/52249/commits/bca6b824781ec0139dd5b5648763de7594f9a6cd)